### DISCUSSION TITLE
rustic don't support command "rewrite" ?

### DISCUSSION THREAD

**Discussion by gdtv** [rustic-rs/rustic#1193] [Q&A] [answered]
restic has a command "rewrite", see the doc : https://restic.readthedocs.io/en/stable/045_working_with_repos.html#removing-files-from-snapshots

It seems it's not supported by rustic ?

**Comment by aawsome** [comment, marked as answer]
Not yet, see https://github.com/rustic-rs/rustic/issues/301.
It is not that much a big deal to implement it, but more about time constraints. If there is anyone up with a PR, we'll love to add that feature.
In meanwhile you should be able to use `restic rewrite` on the rustic repositories in most cases.
